src/util/json.h

```diff
@@ -465,4 +467,18 @@ inline const json_stringt &to_json_string(const jsont &json)
 
 bool operator==(const jsont &left, const jsont &right);
 
+/// Convert the structured_data into an json object. For the example structured
```

Comment:
`structured_data` -> `structured_datat` - also there's no example structured data here now, as this comment has moved from it's original location (sorry...).